spec/services/statistics_service_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.describe StatisticsService do
+  before do
+    # build tournament with predictable test data
+    tournament = create(:tournament)
+    group_stage = create(:group_stage)
+    group = group_stage.groups.first
+    @most_dominant_score = GroupScore.new team: create(:team),
+                                          group_points: 100,
+                                          scored_points: 100, received_points: 0
+    @least_dominant_score = GroupScore.new team: create(:team),
```

Comment:
I think you should use a factory here

Reply:
How should I name it? And what should it contain?
Imo it's not worth the overhead, but if you think so, I will oblige.

Reply:
`:group_score` ?

Reply:
It think it is good just to separate test from test data where possible..